docs/articles/expensify-classic/expense-and-report-features/Attendee-Tracking.md

```diff
@@ -2,4 +2,48 @@
 title: Attendee Tracking
 description: Attendee Tracking
```

Comment:
@laurenreidexpensify Can we do a better description so that it is not the same as the title?

Reply:
updated, sorry I clicked quickly and didn't give the additional commit a different name!